docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -225,20 +225,20 @@ For the list of supported layers, see the **GNA** column of the **Supported Laye
 
 Limitations include:
 
-- Only 1D convolutions are natively supported.
+- Only 1D convolutions are natively supported on the HW prior to GNA 3.0; 2D convolutions have specific limitations (see the table above).
 - The number of output channels for convolutions must be a multiple of 4.
 - The maximum number of filters is 65532 for GNA 2.0 and 8192 for GNA 3.0.
-- Permute layer support is limited to the cases where no data reordering is needed or when reordering is happening for two dimensions, at least one of which is not greater than 8.
+- Transpose layer support is limited to the cases where no data reordering is needed or when reordering is happening for two dimensions, at least one of which is not greater than 8.
```

Comment:
Once the new transpose implementation is merged, this restriction will no longer be accurate.  Please let's merge it ASAP (once tests are written and passing) since it provides a huge performance improvement in some cases.

Reply:
Sure, but this is documentation for 2022.1 which will not have this possibility.